wandb/integration/keras/keras.py

```diff
@@ -48,22 +53,43 @@ def is_generator_like(data):
 
 
 def patch_tf_keras():
+
     from tensorflow.python.eager import context
-    from tensorflow.python.keras.engine import training
-
-    try:
-        from tensorflow.python.keras.engine import training_arrays
-        from tensorflow.python.keras.engine import training_generator
-    except ImportError:
-        from tensorflow.python.keras.engine import training_arrays_v1 as training_arrays
-        from tensorflow.python.keras.engine import (
-            training_generator_v1 as training_generator,
-        )
+
+    from tensorflow import __version__ as tf_version
+
+    if parse_version(tf_version) >= parse_version("2.6.0"):
+        keras_engine = "keras.engine"
```

Comment:
This is a big change.  It would probably be worth updating our unit tests to use 2.6.0+ and be sure everything is still working.  I would rather be testing the newest tensorflow in our regular unit tests, and have the regression continue testing older versions.

Reply:
Yeah... I'm planning to add a few `yea` tests with different version of TF/Keras